README.md

```diff
@@ -215,6 +215,11 @@ In this case you can disable internal webpack:
  ],
 ```
 
+Also you should disable `webpack` option when you use Size Limit for tracking 
+size of CSS files. Otherwise you will get wrong numbers, because webpack inserts 
```

Comment:
We should explain that you should disable Webpack only if you are tracking _only_ CSS files

Reply:
I rehashed it a bit. Check, please.

(And feel free to squash my commits.)